cylc/flow/subprocctx.py

```diff
@@ -155,3 +155,12 @@ def get_signature(self):
         args = self.func_args + [
             "%s=%s" % (i, self.func_kwargs[i]) for i in skeys]
         return "%s(%s)" % (self.func_name, ", ".join([str(a) for a in args]))
+
+    def __str__(self):
```

Comment:
Represents a sub proc context thinggy like this:

`func(arg):interval`

Tested in `coerce_xtrigger`.